src/render_test.js

```diff
@@ -2,7 +2,7 @@
 /* global describe, it, before, beforeEach, afterEach, after */
 
 const fs = require("fs");
-const vm = require("vm");
+const jsdom = require("jsdom");
```

Comment:
The great thing here is that all the test cases are with the react-mount.js in webapp that is currently live on the site, demonstrating that this update still works with that code. Should mean we can update the RRS before we even release the new SSR updates to webapp. Great stuff!